### PR TITLE
Ceramic update

### PR DESCRIPTION
**Pull request by PaulLeCam**

### CODE REVIEW

File: scripts/author.js [hunk 1]
```diff
@@ -20,34 +20,27 @@ if (!process.env.SEED) {
...
-    ceramic,
-    seed: fromString(process.env.SEED),
-    getPermission() {
-      return Promise.resolve([])
-    },
-    disableIDX: true,
-  })
-  const did = new DID({ provider: wallet.getDidProvider() })
+  const seed = fromString(process.env.SEED, 'base16')
+  const did = new DID({ provider: new Ed25519Provider(seed), resolver: KeyResolver.getResolver() })
```
Comment (oed) [inline scripts/author.js:24]: Nice 👍

**Review by oed — COMMENTED** [review]
Looks good from what I can see :)

**Review by oed — APPROVED** [review]

File: scripts/author.js (hunk 1 above)
Comment (PaulLeCam) [reply scripts/author.js:24]: Yeah it's great to be able to do without IDW in this setup now!

**Comment by PaulLeCam** [comment]
@oed I'm not sure how to address the reason it's failing in CI (https://github.com/ceramicstudio/js-idx-tools/runs/1480595754?check_suite_focus=true): for the schemas we want to use the doc version ID, but it seems simply publishing the config (signed records) can affect this? Is this because of anchoring?
Any insight please?

**Comment by oed** [comment]
Looks like a timing issue. You expect DocID with version `0`, but you get the anchored version it seems. Maybe you should create them with `anchor: false`?

**Comment by PaulLeCam** [comment]
Seems all good now, thanks!

**Pull request merged**